src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -1175,30 +1128,16 @@ public async Task WithPreReleaseInvites()
         [Fact]
         public async Task NotLatestPublishedRelease()
         {
-            var publication = new Publication
-            {
-                LatestPublishedReleaseVersion = new ReleaseVersion
-                {
-                    ReleaseName = "2022",
-                    TimePeriodCoverage = TimeIdentifier.CalendarYear,
-                    Published = DateTime.UtcNow
-                }
-            };
+            Publication publication = _dataFixture.DefaultPublication()
+                .WithReleases([_dataFixture.DefaultRelease(publishedVersions: 2)]);
 
-            var notLatestReleaseVersion = new ReleaseVersion
-            {
-                Publication = publication,
-                ReleaseName = "2021",
-                TimePeriodCoverage = TimeIdentifier.CalendarYear,
-                Published = DateTime.UtcNow
-            };
+            var notLatestReleaseVersion = publication.ReleaseVersions[0];
```

Comment:
You could reference this with `publication.Releases[0].Versions[0]` similar to the change you made above.